cmd/cluster-operator/README.md

```diff
@@ -27,13 +27,19 @@ helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRD
 kubectl create namespace kurator-system
 helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
 helm install kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz -n kurator-system
+helm install -f /root/capa-values.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
```

Comment:
The docs should go to docs dir

Reply:
this is a doc for dev, will add doc for cluster operator later.